tests/xfstests/run.pm

```diff
@@ -361,7 +365,7 @@ sub collect_fs_status {
     my ($category, $num) = @_;
     my $cmd = <<END_CMD;
 mount \$TEST_DEV \$TEST_DIR &> /dev/null
-[ -n "\$SCRATCH_DEV" ] && mount \$SCRATCH_DEV /mnt/scratch &> /dev/null
+[ -n "\$SCRATCH_DEV" ] && mount \$SCRATCH_DEV $SCRATCH_FOLDER &> /dev/null
```

Comment:
nit: here is nicely visible, how using upper case for both shell variables (expected) and perl variables is confusing.

Reply:
I see, it's inconvenience during code review. But in vim they are in different color, so I didn't realize this is a problem.